packages/api/src/app-router.ts

```diff
@@ -197,19 +196,6 @@ export default async function makeApp(params: CliArgs) {
     app.use(`/${insecureTestToken}`, insecureTest());
   }
 
-  // Populate Kubernetes getOrchestrators and getBroadcasters is provided
-  if (kubeNamespace) {
-    app.use(
```

Comment:
This unused?

Reply:
Think so. https://livepeer.com/api/region and https://livepeer.com/api/broadcaster both do their own thing.